### PR TITLE
Fixed issue with conditional content in customer email

### PR DESCRIPTION
**Pull request by mislam987**
PR for the conditional content fix on customer emails.

### CODE REVIEW

File: apps/eta/emails/caseworker.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
 {{#data}}
 {{#table}}
 {{#value}}
+  {{label}}
```
Comment (Rhodine-orleans-lindsay) [inline apps/eta/emails/caseworker.html:16]: Did you mean to keep this in?

File: apps/eta/behaviours/customer-email.js [hunk 2]
```diff
@@ -10,8 +10,8 @@ const getDataRows = model => {
     {
       table: [
         model['what-is-your-question-about'] &&  {
-          value: model['what-is-your-question-about'] === 'I-have-not-received-a-confirmation-email' ||
-              model['what-is-your-question-about'] ===  'I-have-not-received-a-decision' ?
+          label: model['what-is-your-question-about'] === 'I have not received a confirmation email' ||
```
Comment (Rhodine-orleans-lindsay) [inline apps/eta/behaviours/customer-email.js:13]: I think needs to be reverted?

**Review by Rhodine-orleans-lindsay — CHANGES_REQUESTED** [review]
looks good just a couple of changes

File: apps/eta/emails/caseworker.html (hunk 1 above)
Comment (mislam987) [reply apps/eta/emails/caseworker.html]: Removed the extra property.

File: apps/eta/behaviours/customer-email.js (hunk 2 above)
Comment (mislam987) [reply apps/eta/behaviours/customer-email.js]: Reverted back to value.

File: apps/eta/views/confirmation.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<title>{{#t}}pages.confirmation.title{{/t}} – GOV.UK</title>
+<title>{{#t}}Question sent - Ask a question about ETA{{/t}} – GOV.UK</title>
```
[the remark this answers is not in the document]
Comment (mislam987) [reply apps/eta/views/confirmation.html:1]: Included the whole title to keep it the same as in the wireframe. Also kept the title in the html page to be consistent with the other pages.

**Review by Rhodine-orleans-lindsay — APPROVED** [review]
lgtm

**Review by shamiluwais — APPROVED** [review]

**Pull request merged**